src/config/SettingsTab.ts

```diff
@@ -78,5 +78,37 @@ export class SettingsTab extends PluginSettingTab {
                         await this.plugin.saveSettings();
                     });
             });
+
+        new Setting(containerEl)
+            .setName('Auto-suggest task content')
+            .setDesc(
+                'Enabling this will open an intelligent suggest window while typing inside a recognized task line',
+            )
```

Comment:
I wonder about adding `in Live Preview  and Source mode.` to the message?

It's great that it works in both.